Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.37018&attempt_id=9afe99bc-5615-43e5-87a1-8773e8e799fa)

Attempted: inkscape

<details><summary>Partial log (click to expand)</summary><p>

```
make[3]: *** [Makefile:751: test-suite.log] Error 1
make[3]: Leaving directory '/build/gsl-2.4/linalg'
make[2]: *** [Makefile:859: check-TESTS] Error 2
make[2]: Leaving directory '/build/gsl-2.4/linalg'
make[1]: *** [Makefile:930: check-am] Error 2
make[1]: Leaving directory '/build/gsl-2.4/linalg'
make: *** [Makefile:961: check-recursive] Error 1
builder for '/nix/store/gpn328f98y13xvh3rkzrqdyfk8m7z8qj-gsl-2.4.drv' failed with exit code 2
cannot build derivation '/nix/store/4mxgvvlj0xbxk25i1g0igpiq0arp79s8-inkscape-0.92.3.drv': 1 dependencies couldn't be built
[31;1merror:[0m build of '/nix/store/4mxgvvlj0xbxk25i1g0igpiq0arp79s8-inkscape-0.92.3.drv' failed
```
</p></details>

--- a/pkgs/applications/graphics/inkscape/default.nix
+++ b/pkgs/applications/graphics/inkscape/default.nix
@@ -10,11 +10,11 @@ let
 in
 
 stdenv.mkDerivation rec {
-  name = "inkscape-0.92.2";
+  name = "inkscape-0.92.3";
 
   src = fetchurl {
     url = "https://media.inkscape.org/dl/resources/file/${name}.tar.bz2";
-    sha256 = "1lyghk6yarcv9nwkh6k366p6hb7rfilqcvbyji09hki59khd0a56";
+    sha256 = "1chng2yw8dsjxc9gf92aqv7plj11cav8ax321wmakmv5bb09cch6";
   };
 
   unpackPhase = ''

